pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -543,16 +544,30 @@ func (ds *DistSender) singleRangeFeed(
 
 		log.VEventf(ctx, 3, "attempting to create a RangeFeed over replica %s", args.Replica)
 
+		ctx := ds.AnnotateCtx(ctx)
+		ctx = logtags.AddTag(ctx, "dest_n", args.Replica.NodeID)
+		ctx = logtags.AddTag(ctx, "dest_s", args.Replica.StoreID)
+		ctx = logtags.AddTag(ctx, "dest_r", args.RangeID)
```

Comment:
nit: the `dest_` prefix may not be necessary, since it should be clear from the context. No strong opinion though.